.pre-commit-config.yaml

```
@@ -0,0 +1,56 @@
+exclude: Pipfile\.lock|migrations|\.idea
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: trailing-whitespace
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-case-conflict
+      - id: check-toml
+      - id: check-merge-conflict
+      - id: check-xml
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: check-symlinks
+      - id: mixed-line-ending
+      - id: fix-encoding-pragma
+        args:
+          - --remove
+      - id: pretty-format-json
+        args:
+          - --autofix
+      - id: requirements-txt-fixer
+
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.9.1
+    hooks:
+      - id: isort
+        name: Run isort to sort imports
+        files: \.py$
+        exclude: ^build/.*$|^.tox/.*$|^venv/.*$
+        args:
+          - --lines-after-imports=2
+          - --multi-line=3
+          - --trailing-comma
+          - --force-grid-wrap=0
+          - --use-parentheses
+          - --ensure-newline-before-comments
+          - --line-length=88
+
+  - repo: https://gitlab.com/pycqa/flake8
```

Just to match the official way their org is named.
```suggestion
  - repo: https://gitlab.com/PyCQA/flake8
```

---

Their [docs](https://flake8.pycqa.org/en/latest/user/using-hooks.html) show a GitHub URL. Let's use that.
```suggestion
  - repo: https://github.com/PyCQA/flake8
```